### PR TITLE
Updated Spanish translations from en file

### PR DESCRIPTION
**Pull request by natastro**
Translated en.json using a mix of what was on es.json and translating the new lines. In reference to #64 

Please let me know if I need to change anything or if you need anything else :) Thank you!!

### CODE REVIEW

File: resources/lang/es.json [hunk 1]
```diff
@@ -73,17 +99,145 @@
...
+  "manual_use_thumbs": "Si está habilitada, las miniaturas se utilizan para la galería independiente. Esto mejorará ligeramente el rendimiento.",
+  "manual_user_interface_background_admin": "Estilo CSS para fondo en el panel de administración.",
+  "manual_user_interface_background_chroma": "Estilo CSS para fondo en la página de cromakeying.",
+  "manual_user_interface_background_image": "Estilo CSS para fondo en páginas de inicio, inicio de sesión y presentación de diapositivas.",
+  "manual_user_interface_font_size": "Introduzca el tamaño de fuente predeterminado utilizado para la interfaz Photobooth.",
+  "milliseconds": "los milisegundos",
```
Comment (andi34) [inline resources/lang/es.json]: `milisegundos` should be enough If I remember right from school.

File: resources/lang/es.json [hunk 2]
```diff
@@ -93,16 +247,23 @@
...
+  "remove_config": "Eliminar configuración personal (my.config.inc.php)",
+  "remove_images": "Eliminar imágenes",
+  "remove_mailtxt": "Eliminar la base de datos de direcciones de correo electrónico",
   "reset": "Reiniciar",
+  "rounded_corners": "Esquinas redondeadas",
   "save": "Salvar",
   "saveerror": "Error",
   "saving": "En proceso de copia de seguridad.",
   "scrollbar": "Muestre la barra de desplazamiento en la galería",
+  "seconds": "los segundos",
```
Comment (andi34) [inline resources/lang/es.json]: Same here, `sugundo(s)`

Comment (andi34) [reply resources/lang/es.json]: Example from countdown time inside Adminpanel:
`10 los segundos` vs `10 segundo(s)`

**Review by andi34 — CHANGES_REQUESTED** [review]
Thanks for your contribution! I think some small parts should be changed. Not sure if I am right, see comments. I've started learning Spanish years ago but stopped with it once I was out of school... long time ago 🙈

File: resources/lang/es.json (hunk 1 above)
Comment (natastro) [reply resources/lang/es.json]: Changed! I'll be submitting the updated pull request :)

File: resources/lang/es.json (hunk 2 above)
Comment (natastro) [reply resources/lang/es.json]: Right again :) I changed it to `segundo(s)` if that's okay

**Comment by natastro** [comment]
Hi @andi34 ! Looks like the new commit went through. Please let me know if anything else is needed and thank you for your input!

**Comment by andi34** [comment]
Thanks @stroudn1!
After https://github.com/andi34/photobooth/pull/70 there's 2 more to translate. You'd like to look at those 2 too?

**Comment by natastro** [comment]
@andi34 Absolutely! Can I begin translating now or should I wait?

**Comment by andi34** [comment]
Great! You could commit your changes to the `feature/qr-ssid` and create a PR for same branch, this would allow me to merge it to the main branch in one "topic".

**Comment by natastro** [comment]
Thank you @andi34 :) I was looking at en.json - is this where the 2 new ones are? Sorry for the confusion

**Comment by andi34** [comment]
Yeah it is. 

qrHelp was adjusted,
```
"qrHelp": "To download the picture to your smartphone, connect to the WiFi:",
```

2 new:
```
"general_wifi_ssid": "Wireless network name (SSID) used to access the photobooth",
 "manual_general_wifi_ssid": "Please define the wireless network name (SSID) to be used to access the Photobooth. The wireless network name (SSID) is displayed on the results page when the QR code is called up.",
```

**Pull request merged**